questionnaires/superset/README.md

```diff
@@ -112,3 +112,46 @@ docker-compose up
 ```
 docker-compose up
 ```
+
+## Share Dashboard
+
+### How to Share Dashboard Publicly
+
+Anyone with the link will be able to view the dashboard.
```

Comment:
All data in this dataset should be considered viewable publicly. If someone has the URL to your dashboard, they will be able to view your visualizations without a Superset account.